docs/notebooks/WMD_tutorial.py

```diff
@@ -0,0 +1,329 @@
+
+# coding: utf-8
+"""
+WMD_tutorial.py
+===================
+
+
+.. _general_examples:
+
+General examples
+----------------
+
+General-purpose and introductory examples from the sphinx-gallery
+"""
```

Comment:
@tmylk Sphinx-gallery processes py files only. Moreover as you can see above, each file should have a "title" and a "description" which is the part that I manually added. Also the notebooks had to be edited because of various py3 compatibility issues.